addons/web_editor/static/lib/odoo-editor/test/spec/list.test.js

```diff
@@ -94,6 +94,13 @@ describe('List', () => {
                             `),
                         });
                     });
+                    it('should create a new unordered list if closestBlock is deeply nested inside a list', async () => {
+                        await testEditor(BasicEditor, {
+                            contentBefore: '<ul><li><div><p>a[]b</p></div></li></ul>',
+                            stepFunction: toggleUnorderedList,
+                            contentAfter: '<ul><li><div><ul><li>a[]b</li></ul></div></li></ul>',
```

Comment:
Is that really what we want? That seems weird to me: I'm in a list, I ask to toggle list, I don't want to get a nested list. How do I remove the top list then?

This seems like an ad-hoc fix that is supposed to be just for the mega menu, where I guess we don't ever want to remove the top list. In that case it would make more sense to just not count `li.nav-item` as a list item and its parent as a list.

cc @sebgeelen